Backend/controllers/adminc.js

```diff
@@ -0,0 +1,16 @@
+let admin = [];
+const createAdmin = async (req, res) => {
+  const { email, password } = req.body;
+  admin.push(email);
```

Comment:
You should try to add email and password validation in the server code as well. And instead of storing the password directly, encrypt it. Although I presume that the encryption aspect is to be implemented later on, though do keep this in mind